lib/ret_web/email.ex

```diff
@@ -0,0 +1,19 @@
+defmodule RetWeb.Email do
+  use Bamboo.Phoenix, view: RetWeb.EmailView
+
+  def auth_email(to_address, signin_args) do
+    new_email()
+    |> to(to_address)
+    |> from(from_address())
```

Comment:
We probably want to format the from field as "Mozilla Hubs <info@hubs-mail.com>". I think Bamboo can take a `{name, address}` tuple here, if I'm reading their docs correctly.